agm.html

```diff
@@ -0,0 +1,78 @@
+              <ul>
+                <li>Chair's report</li>
+                <li>Treasurer's report</li>
+                <li>Sysadmins' report</li>
+              </ul>
+            </li>
+            <li>Consideration of received motions, if any</li>
+            <li>Election procedure
+              <ul>
+                <li>Election candidates' speeches</li>
+                <li>Voting (procedure TBD)</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="card">
+    <div class="card-body">
+      <h4 class="card-title">Call for Submissions</h4>
+      <p>If you wish to stand for election to the committee, please read the dedicated section below.</p>
+      <p>If you wish to submit a motion for the AGM, please contact the Secretary at least three days in advance of the meeting date with details of the motion.  Examples of types of motion are resolutions to adopt a policy, mandates to the SRCF committee, or constitutional amendments.</p>
+    </div>
+  </div>
+
+  <div class="card">
+    <div class="card-body">
+      <h4 class="card-title">Committee Elections</h4>
+      <p>The AGM will include the election of the SRCF Committee for 2021-22.</p>
+      <p>We strongly encourage anyone interested, regardless of prior involvement with the SRCF, to nominate themselves for a committee position. All committee positions have a low time commitment and are accessible for those with any amount of experience. It's an incredibly valuable experience, and you'll be contributing to a society that is so crucial to hundreds of our fellow societies.</p>
+      <p>Rules on the election of the SRCF Committee are contained in Clause 5 of <a href="{{ DOMAIN_WEB }}/constitution">the constitution </a> and the most important points are reproduced below:</p>
+        <ul>
+          <li>"Nominations must be submitted to the Secretary at least three days before the AGM (in the absence of other nominations for any post, nominations shall be accepted at the AGM for that post), either by email or on paper, to reach the Secretary by that deadline signed by proposer, seconder and nominee."</li>
+          <li>"Any member of the society may be nominated for any Committee post (a proposer and seconder are needed). The Chair of the AGM shall appoint two tellers at the AGM."</li>
+        </ul>
+      <p>If you are interested in running for any of the above positions, but you want to know more about what is involved, you might want to read our page about <a href="{{ DOMAIN_WEB }}/roles">roles in the SRCF</a>.</p>
+    </div>
+  </div>
+
+  <div class="card">
+    <div class="card-body">
+      <h4 class="card-title">Other Notes</h4>
+      <ul>
+        <li>Do not let the proposer and seconder requirement discourage you! If you send in a short manifesto/blurb explaining your motivation for standing for the position, the committee will be happy to propose/second you.</li>
```

Comment:
Hoist this sentence up to line 61 (i.e. make this part of the existing last paragraph of previous section)!